.travis.yml

```diff
@@ -14,9 +14,5 @@ before script:
   - psql -c "create database ahsummerdb;" -U postgres
 
 script:
-  - python manage.py test
-  - coverage
-  - coveralls
-
-after success:
+  - coverage run --omit="*wsgi*,*production*," --source="authors" manage.py test&&coverage report -m
```

Comment:
Good work finally setting this up properly @codjoero. I would suggest you separate these two commands though and instead of combining them with the `&&`, you'd just move `coverage report -m` to be a separate command on the next line, to reduce the complexity of that line.

Reply:
Thanks, @Karuhanga let me look into it.